protocol-definitions/Sql.yaml

```diff
@@ -177,14 +177,14 @@ methods:
               ANY(0)
               ROWS(1)
               UPDATE_COUNT(2)
-    response:
-      params:
         - name: queryId
           type: SqlQueryId
           nullable: true
```

Comment:
@devozerov I think this field is mandatory and can not be `nullable`, wdyt ?

Reply:
Yes, it was a bug in 4.1, that I copied blindly to the new message. Fixed in https://github.com/hazelcast/hazelcast-client-protocol/pull/366/commits/993d4a89b25971ea9120471674c320d389a4c030